### ISSUE TITLE
msg.payload.buienradar does not contain all fields returned by the buienradar xml response

### ISSUE THREAD

**Issue by thomasvnl** [dkijkuit/node-red-contrib-buienradar#5]
Hi there,

I think its a pitty that your great piece of node-red-contrib-buienradar does not return all of the fields that are returned by the XML response from the buienradar api.

For instance, you do not return the _zonintensiteitWM2_, nor the _windsnelheidMS_ fields, even though there is no reason not to and they could be useful for others too. Could you include the rest of these fields in your node-red function please?

With kind regards,
Thomas

The full XML response (per measurement station) for reference.
```
<weerstation id="6391">
  <stationcode>6391</stationcode>
  <stationnaam regio="Venlo">Meetstation Arcen</stationnaam>
  <lat>51.50</lat>
  <lon>6.20</lon>
  <datum>05/30/2018 21:20:00</datum>
  <luchtvochtigheid>87</luchtvochtigheid>
  <temperatuurGC>21.9</temperatuurGC>
  <windsnelheidMS>0.14</windsnelheidMS>
  <windsnelheidBF>0</windsnelheidBF>
  <windrichtingGR>277</windrichtingGR>
  <windrichting>W</windrichting>
  <luchtdruk>-</luchtdruk>
  <zichtmeters>-</zichtmeters>
  <windstotenMS>1</windstotenMS>
  <regenMMPU>-</regenMMPU>
  <zonintensiteitWM2>9</zonintensiteitWM2>
  <icoonactueel ID="c" zin="Zwaar bewolkt">
    https://www.buienradar.nl/resources/images/icons/weather/30x30/c.png
  </icoonactueel>
  <temperatuur10cm>20.8</temperatuur10cm>
  <url>
    http://www.buienradar.nl/nederland/weerbericht/weergrafieken/6391
  </url>
  <latGraden>51.83</latGraden>
  <lonGraden>6.33</lonGraden>
</weerstation>
```

**Comment by dkijkuit** [comment]
Implemented with commit d889018d97eb9f5a875cabc53991e7b4963add43

**Comment by thomasvnl** [comment]
Thanks David, much appreciated.

Thomas
